Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -2778,6 +2778,7 @@ void CClientGame::AddBuiltInEvents()
     m_Events.AddEvent("onClientBrowserTooltip", "text", NULL, false);
     m_Events.AddEvent("onClientBrowserInputFocusChanged", "gainedfocus", NULL, false);
     m_Events.AddEvent("onClientBrowserResourceBlocked", "url, domain, reason", NULL, false);
+    m_Events.AddEvent("onClientBrowserConsoleMessage", "message, source, line, level", NULL, false);
```

Comment:
Change NULL to nullptr

Reply:
it doesn't change anything, it's like this everywhere, you can open pull request to refactor mta code

Reply:
> it doesn't change anything, it's like this everywhere, you can open pull request to refactor mta code

It changes everything. Why should someone open a pull request just to clean after you?

Reply:
> it doesn't change anything, it's like this everywhere, you can open pull request to refactor mta code

In accordance with the guidelines and current C++ conventions, you should use nullptr instead of NULL, as everyone else does in this repository. No one is going to clean up after you because you're too lazy to make the changes required by the coding standards. We follow the principle that when creating new code, we adhere to the latest guidelines and avoid creating more mess than there already is. If you don't follow the coding guidelines, your PR simply won't be considered, and no one will likely want to merge it. It's your choice whether to spend a minute making this change or risk having all the time you invested in this PR go to waste